src/data/markdown/docs/30 xk6-browser/02 API/03 BrowserType/launch--options--.md

```diff
@@ -13,12 +13,58 @@ Launches a new browser process.
 | executablePath    | string   | `null`  | Override search for browser executable in favor of specified absolute path.                                                                                                                                                                                           |
 | headless          | boolean  | `true`  | Show browser GUI or not.                                                                                                                                                                                                                                              |
-| ignoreDefaultArgs | string[] | `null`  | Ignore any of the default arguments included when launching browser process.                                                                                                                                                                                          |
+| ignoreDefaultArgs | string[] | `null`  | Ignore any of the [default arguments](#default-arguments) included when launching browser process.                                                                                                                                                                                          |
 | proxy             | string   | `null`  | Specify to set browser's proxy configuration.                                                                                                                                                                                                                         |
 | slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
 | timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
 
 
+### Default arguments
+
+List of default arguments included when launching the browser process.
+
+**Note:** The starting '--' have been omitted from the argument names in these lists.
+
+| Argument | Value  | Description                                            |
+|----------|--------|--------------------------------------------------------|
+| disable-background-networking | `true` | Disables several subsystems which run network requests in the background. This is for use when doing network performance testing to avoid noise in the measurements. |
```

Comment:
```suggestion
| disable-background-networking | `true` | Disables several subsystems which run network requests in the background. This is used during network performance testing to avoid measurement noise. |
```

Reply:
Applying.